ethcore/sync/src/chain/handler.rs

```diff
@@ -625,10 +672,12 @@ impl SyncHandler {
 			asking_blocks: Vec::new(),
 			asking_hash: None,
 			ask_time: Instant::now(),
+			ask_bitfield_time: Instant::now() - BITFIELD_REFRESH_DURATION,
```

Comment:
Is it safe to subtract `BITFIELD_REFRESH_DURATION` from `Instant::now()` and why is it needed?